# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/webrender_bindings/RenderCompositorEGL.cpp
# Commit: c3b9163075af
# Full Hash: c3b9163075afc6e46a3f96c72b10fcc5e8e2ab93
# Author: stransky <stransky@redhat.com>
# Date: 2024-12-17 17:00:49
# Regressor Bug: 1934497
# File Overlap Count: 1
# Description:
#   Bug 1934497 [Wayland] Switch MozContainerSurfaceLock for WaylandSurfaceClock at RenderCompositorEGL r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230854
# ==============================================================================

diff -r 4e75ec1344b9 -r c3b9163075af gfx/webrender_bindings/RenderCompositorEGL.cpp
--- a/gfx/webrender_bindings/RenderCompositorEGL.cpp	Tue Dec 17 06:00:02 2024 +0000
+++ b/gfx/webrender_bindings/RenderCompositorEGL.cpp	Tue Dec 17 06:00:02 2024 +0000
@@ -159,7 +159,7 @@
   // Rendering on Wayland has to be atomic (buffer attach + commit) and
   // wayland surface is also used by main thread so lock it before
   // we paint at SwapBuffers().
-  UniquePtr<MozContainerSurfaceLock> lock;
+  UniquePtr<widget::WaylandSurfaceLock> lock;
   if (auto* gtkWidget = mWidget->AsGTK()) {
     lock = gtkWidget->LockSurface();
   }
